specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -386,15 +484,15 @@ subscriptions/{subId}/resourceGroups/{rg}/providers/Microsoft.LoadTestService/lo
 
   @doc("The creation datetime(ISO 8601 literal format).")
   @visibility("read")
-  createdDateTime?: offsetDateTime;
+  createdDateTime?: utcDateTime;
 
   @doc("The user that created.")
   @visibility("read")
   createdBy?: string;
 
   @doc("The last Modified datetime(ISO 8601 literal format).")
   @visibility("read")
-  lastModifiedDateTime?: offsetDateTime;
+  lastModifiedDateTime?: utcDateTime;
 
   @doc("The user that last modified.")
   @visibility("read")
```

Comment:
why are resourceId etc nullable inside an appComponent?

Reply:
Updated resourceId to be required and read only field. Updated other also after discussion with Krishna